hugegraph-server/hugegraph-core/src/main/java/org/apache/hugegraph/StandardHugeGraph.java

```diff
@@ -121,26 +121,26 @@
 public class StandardHugeGraph implements HugeGraph {
 
     public static final Class<?>[] PROTECT_CLASSES = {
-        StandardHugeGraph.class,
-        StandardHugeGraph.StandardHugeGraphParams.class,
-        TinkerPopTransaction.class,
-        StandardHugeGraph.Txs.class,
-        StandardHugeGraph.SysTransaction.class
+            StandardHugeGraph.class,
+            StandardHugeGraph.StandardHugeGraphParams.class,
+            TinkerPopTransaction.class,
+            StandardHugeGraph.Txs.class,
+            StandardHugeGraph.SysTransaction.class
```

Comment:
**IMPORTANT:** we need to ensure 4 or 8 spaces to format (and sync to "hugegraph-style" **config files** & other PRs)